.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -0,0 +1,5 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
Is it useful to add these new files under .idea? If so, I would consider adding them in a separate PR.

Reply:
Will remove, looks to have snuck into my commit :)